src/OpenSage.Game/Data/Apt/Characters/Button.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.IO;
+using System.Numerics;
+using OpenSage.Data.Utilities.Extensions;
+using OpenSage.Mathematics;
+
+namespace OpenSage.Data.Apt.Characters
+{
+    enum ButtonState
+    {
+        Idle = 0,
+        OutUp = 1,
+        OverUp = 2,
+        OutDown = 3,
+        OverDown = 4,
+    }
+
+    public class Button : Character
+    {
+        public Vector4 Bounds { get; private set; }
+        public IndexedTriangle[] Triangles { get; private set; }
+        public Vector2[] Vertices { get; private set; }
+
+        public static Button Parse(BinaryReader reader)
+        {
+            var b = new Button();
```

Comment:
I'd prefer if you used full words for local variable names, like `button` here. I know there are several different schools of thought on this, but that's what I've done in the rest of the code.